cnv_germline_case_scattered_workflow.wdl

```diff
@@ -0,0 +1,250 @@
+#
+# A wrapper for the gCNV case workflow intended for lowering computing cost by making it feasible to use 
+# preemptible cloud instances with low memory requirements. CPU, memory and disk requirements can be 
```

Comment:
This WDL is only used to do single sample analysis of large cohorts, it reduces price and makes it feasible to run > few hundred samples. I think we had conversation about whether this belongs in the featured workspace or not. I feel like it might confuse the users, and it is also easy to write this wrapper if needed. @ldgauthier @samuelklee What do you guys think?

Reply:
Let's leave it out for this version.  It might confuse users and it's soooo painful to set up in the data model (unless someone has a better way than what I've done.)

Reply:
k, will remove

Reply:
Sorry to come late to the discussion. I see that this has already been removed, which is fine for this version.  But I think it ultimately makes sense for this to be featured and the non-scattered-case workflow to be hidden as a subworkflow.  Certainly if we publish a certain price per sample and advertise the ability to run large cohorts in the upcoming paper, let's make sure this is in shape by then.  The input data model is the same as for cohort mode, and it's probably not too much work to clean up the output model.